cleaning up code and removed unncessary files

jadonx opened the pull request:
Before issuing a pull request, please see the contributing page.

KAnuje committed:
Cleaning up files

KAnuje committed:
Adding Cougar Robot class and its instantiation in another class

jadonx committed:
Field Centric Movement

KAnuje committed:
Fixing FieldCentric

jadonx committed:
cleaned up unnecessary code

jadonx committed:
Merge branch 'main' of https://github.com/FTC226/into_the_deep

jadonx committed:
fixed branch difference error

jadonx committed:
CougarRobot move method works on official robot

FTC226 commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/CougarRobot.java:
```diff
@@ -0,0 +1,259 @@
+package org.firstinspires.ftc.teamcode;
```
@jadonx I would suggest using a different name for this file, `CougarRobot` doesn't tell me what is different about this file vs others - is this for a field-centric OpMode? Autonomous?

Is this the code from 2023?

FTC226 commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotMovementFieldCentric.java:
```diff
@@ -54,7 +54,7 @@ public void runOpMode() {
             double rotY = x * Math.sin(-botHeading) + y * Math.cos(-botHeading);
 
-            rotX = rotX * 1.1;  // Counteract imperfect strafing
+            rotX = rotX * 1.4;  // Counteract imperfect strafing
```
2 comments:

1. Consider making this a variable and allow the variable to be changed via buttons (maybe A/B, etc) so the driver can change the value while the program is running and tune the setting. This means you would not be redownloading the program everytime you change / guess this value. Once the driver has done some trial and error, you can then preset the value to whatever they have found to be the best. Note: you'll need to output the variable value via telemetry
2. Any thoughts on why we have imperfect strafing? Is it due to weight distribution? Something else? We should try to run some experiments at some point to identify the source of this problem.

jadonx committed:
9/10 meeting, created updateHeading function for field centric

jadonx committed:
Created program that test IMU and field centric

jadonx committed:
added wheel test method

jadonx committed:
teleop linear slide working

jadonx committed:
field centric works, linear slide teleop works

jadonx committed:
changed readme

jadonx committed:
changed reamde

jadonx committed:
chagned readme #3

jadonx committed:
fixed readme

jadonx committed:
9/19 meeting: field centric, camera, linearslides are working

jadonx committed:
organized java files into testing and final programs

KAnuje committed:
making kunal branch and creating class for first auto

KAnuje committed:
making sure it works

KAnuje committed:
Remaking the packages and refactoring

KAnuje committed:
created an auto with a moving with spline (moveToSpline()) and moving while rotating (moveTo()) method

KAnuje committed:
Adding Object Recognition to the CameraTest.java with a method to display the distance away from the camera.

KAnuje committed:
Changing Linear Slide test - this works using runToPosition, rather than the power

KAnuje committed:
Adding Movement Tester to try to fix the turning problem during FC
1. EncoderTest.java
2. straightTest.java
3. straightFCTest.java (along with FieldCentricModified.java)

Also adding tester for the claw: only class is clawMovement.java

KAnuje committed:
Added Roadrunner to the project so that we can start auto and added a class (Hand.java) for instantiating in MainDrive when ready for the hand

Z-BUSTER committed:
Merge pull request #3 from FTC226/michael

Michael

Z-BUSTER committed:
claw stuff

Z-BUSTER committed:
VISION CAMERA

Z-BUSTER committed:
Arm stuff

Z-BUSTER committed:
Arm stuff more

Z-BUSTER committed:
Arm stuff more...

Z-BUSTER committed:
Merged Code

Z-BUSTER committed:
Merged Code

Z-BUSTER committed:
Camera movement

Z-BUSTER committed:
stuff

Z-BUSTER committed:
stuff2

Z-BUSTER committed:
stuff3

Z-BUSTER committed:
stuff4

Z-BUSTER committed:
stuff5

Z-BUSTER committed:
Subsets for claw arm and slides not really working

Z-BUSTER committed:
Tried to get slides to work

Z-BUSTER committed:
Got slides to work

Z-BUSTER committed:
Did a bunch of stuff

Z-BUSTER committed:
Merge pull request #6 from FTC226/michael4

Michael4

Z-BUSTER committed:
Tried to get telop to work

Z-BUSTER committed:
backup

Z-BUSTER committed:
Merge pull request #7 from FTC226/michael4

Michael4

jadonx committed:
updated to thai-duongs branch

jadonx committed:
Merge pull request #9 from FTC226/jadon-worlds

updated to thai-duongs branch

jadonx committed:
added 4 sample auto w/ 12 seconds left and new tuning values

jadonx committed:
removed specimen from teleop

Z-BUSTER committed:
Testing

Z-BUSTER committed:
Merge pull request #11 from Z-BUSTER/testing

Testing

Z-BUSTER committed:
Basic motor stuff

Z-BUSTER committed:
chatgpt does NOT usderstand inverse kinematics at all

Z-BUSTER committed:
Merge pull request #12 from Z-BUSTER/testing

Testing
